packages/flame/test/components/component_test.dart

```diff
@@ -472,3 +541,13 @@ class TwoChildrenComponent extends Component {
     add(child2);
   }
 }
+
+class _ComponentWithOnLoad extends Component {
+  int onLoadCalledCount = 0;
+
+  @override
+  Future<void> onLoad() async {
+    onLoadCalledCount++;
+    await null;
```

Comment:
I don't think this is needed, just by marking the method as async should be enough
```suggestion
```

Reply:
Yep, makes sense.